packages/compass-connections-navigation/src/base-navigation-item.tsx

```diff
@@ -88,7 +90,7 @@ export const NavigationBaseItem = ({
           </ItemLabel>
         </ItemButtonWrapper>
         <ItemActionControls<Actions>
-          isVisible={isActive || isHovered}
+          isVisible={isActive || isHovered || isFocused}
```

Comment:
So this is the main change that makes the action controls tabbable because we render the action controls also when the navigation item is focused.

Reply:
[How we get isFocused](https://github.com/mongodb-js/compass/pull/5923/files#diff-f06442f5a7bdda153a9bab3f55177d9b4165e82f71e7ba13064dddf5ac63ad67R306-R308) is something you guys might wanna comment about.

Reply:
Do you have concerns about this approach?